pandas/tests/extension/base/reduce.py

```diff
@@ -68,3 +68,11 @@ def test_reduce_series(self, data, all_boolean_reductions, skipna):
         op_name = all_boolean_reductions
         s = pd.Series(data)
         self.check_reduce(s, op_name, skipna)
+
+
+class BaseArgReduceTests(BaseReduceTests):
```

Comment:
maybe  instead of creating *another* reduce class (the more we have the harder it is to have consistency), and put it under the indexing umbrella as another test.

Reply:
Where is indexing, do you mean `pandas/tests/extension/base/method.py` ?

Reply:
yeah do it there; we already have some argmin/max tests so add below

Reply:
OK.